tests/fixtures/saas/zendesk_fixtures.py

```diff
@@ -40,7 +40,7 @@ def zendesk_erasure_identity_email() -> str:
 class ZendeskClient:
     def __init__(self, secrets: Dict[str, Any]):
         self.base_url = f"https://{secrets['domain']}"
-        self.auth = secrets["username"], secrets["api_key"]
+        self.auth = secrets["username"] + "/token", secrets["api_key"]
```

Comment:
I commented on this somewhere else too, but if we don't have a migration in place to migrate the username, we should just leave things as they are. I agree this is cleaner, but we would break existing integrations

Reply:
In that case, i think we should add it to the hoverable description on the config.  Something that points to the user that they hadd to add the "/token" at the end without lookinv over the Zendesk Docs